### DISCUSSION TITLE
css-hack-visual-studio-code-move-debug-toolbar/

### DISCUSSION THREAD

**Discussion by giscus** [estruyf/eliostruyf-comments#612] [Comments]
# css-hack-visual-studio-code-move-debug-toolbar/

Azure / Microsoft 365 / Visual Studio Code / Speaker / Blogger

https://www.eliostruyf.com/css-hack-visual-studio-code-move-debug-toolbar/

**Comment by bendera** [comment]
You don't need to hack VSCode. There is a configuration setting called "Debug: Tool Bar Location" and the docked mode you are looking for. I only found it recently.

**Reply by estruyf** [reply]
I know about that setting, but the docked mode moves the controls to the debug panel only, where I mostly have other panels open.

**Reply by bendera** [reply]
I see, that makes sense.

**Comment by kurt-west** [comment]
Thank you for this!! Hopefully one day the VSCode team makes this a standard option, its the right place for it IMO.
